src/library/scala/util/Properties.scala

```diff
@@ -13,6 +13,28 @@ package util
 import java.io.{ IOException, PrintWriter }
 import java.util.jar.Attributes.{ Name => AttributeName }
 
+sealed abstract class JavaSpecVersion extends Ordered[JavaSpecVersion] {
```

Comment:
Well, yes, but the combination of the bits is what is interesting.  In this case, both sets of bits represent integers, and it's normal integer comparison that is wanted.